### PR TITLE
doc: update releases to the current state

### PR DESCRIPTION
**Pull request by theanalyst**
Adding Luminous release date, and dropping links for dev releases (since
they've been merged to 12.2.0)

Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

### CODE REVIEW

**Comment by smithfarm** [comment]
@theanalyst Depending on how it renders, you might consider dropping the Dumpling and Emperor columns?

**Comment by theanalyst** [comment]
jenkins render docs

**Comment by smithfarm** [comment]
The render looks OK even on my non-wide monitor, but I still question whether we need Dumpling and Emperor.

@theanalyst nit: in the Development/Testing column, August 2017, the `12.1.*` releases should be in reverse chronological order to match the rest of the table.

**Comment by theanalyst** [comment]
I guess it makes sense to drop off Dumpling & emperor (even firefly, but since it was sort of the first LTS, maybe it can stay) @liewegas thoughts?

**Comment by smithfarm** [comment]
Also, is it time to declare hammer and kraken EOL?

**Comment by liewegas** [comment]
I think the old release timing is historically interesting.  Maybe if we flip things left to right it won't matter that the chart is big?

**Comment by smithfarm** [comment]
Good idea, @liewegas - flipping it works for me!

**Comment by theanalyst** [comment]
agree, lets flip :)

**Comment by theanalyst** [comment]
@liewegas @smithfarm updated

**Comment by theanalyst** [comment]
jenkins render docs

**Comment by theanalyst** [comment]
(also orgtbl-mode FTW)

**Comment by smithfarm** [comment]
@theanalyst Can you change "Luminous stable" heading to "Luminous LTS" ?

**Comment by smithfarm** [comment]
Also, Giant is on left of Hammer, should be on the right.

**Comment by theanalyst** [comment]
@smithfarm updated

**Comment by smithfarm** [comment]
Jenkins render docs

File: doc/releases.rst [hunk 1]
```diff
@@ -5,173 +5,173 @@ Ceph Releases
...
-.. _12.0.0: ../release-notes#v12-0-0-luminous-dev
++----------------------+-------------+-----------+----------+---------------+-----------+----------+------------+------------+-------------+
+|                      | `Luminous`_ | `Kraken`_ | `Jewel`_ | `Infernalis`_ | `Hammer`_ | `Giant`_ | `Firefly`_ | `Emperor`_ | `Dumpling`_ |
+|                      |  LTS        | Stable    | LTS      | Stable        | LTS       | Stable   | LTS        | Stable     | LTS         |
++----------------------+-------------+-----------+----------+---------------+-----------+----------+------------+------------+-------------+
+| First release        | August      | January   | April    | November      | April     | October  | May        | November   | August      |
+|                      | 2017        | 2017      | 2016     | 2015          | 2015      | 2014     | 2014       | 2013       | 2013        |
++----------------------+-------------+-----------+----------+---------------+-----------+----------+------------+------------+-------------+
+| Estimated retirement |             | August    | November |               | May       |          | January    |            | March       |
```
Comment (dillaman) [inline doc/releases.rst]: Move Infernalis' "August 2017" down to the next row for "Actual Retirement" -- also add "August 2017" for Hammer under the "Actual Retirement" row

Comment (theanalyst) [reply doc/releases.rst]: ack, does this actually warrant a ceph-users/devel announcement?

Comment (dillaman) [reply doc/releases.rst]: Probably would be nice even though this page states that only the most recent two LTSs will be maintained -- @liewegas?

Comment (smithfarm) [reply doc/releases.rst]: @dillaman You meant Kraken, not Infernalis.

Comment (smithfarm) [reply doc/releases.rst]: I don't remember us ever announcing when a release went EOL, but that doesn't mean we didn't :-/

Comment (dillaman) [reply doc/releases.rst]: @smithfarm Indeed -- thanks for keeping me honest. :-p

Comment (theanalyst) [reply doc/releases.rst]: let us start now then :) we announced that hammer would be EOL when L is released, we didn't explicitly announce retirals so far (Back when backports started Loic did one for Dumpling iirc)

Comment (smithfarm) [reply doc/releases.rst]: The best place for the hammer, kraken EOL announcement is/would have been in the Luminous release announcement . . .

Comment (liewegas) [reply doc/releases.rst]: Yeah I don't think we have been sending EOL announcements, but we should probably start :).

A separate email seems better to me (with EOL in the subject) to catch people's attention. (Next time we can *also* mention it in the LTS release too, but whatever.)

**Comment by liewegas** [comment]
Merging this as is so it gets pushed.  We can update EOL dates in a new pr

**Pull request merged**